docs/dns-client/configuration.md

```diff
@@ -38,6 +38,28 @@ The `cache` object configures caching the results of querying DNS. It has the fo
 
 The `server` object configures the handling of incoming requests. It has the following properties:
 
+  - `bind_retry`: The confguration of the retry mechanism for binding to the listen addresses.
```

Comment:
This needs to be expanded regarding why this is necessary at all (and when it may not be necessary).